src/boot/efi/boot.c

```diff
@@ -2380,6 +2527,9 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
 
         config_load_defaults(&config, root_dir);
 
+        Print(L"Checking for secure boot auto enrollement\n");
```

Comment:
This message needs to go into secure_boot_discover_keys() inside the auto_enroll if.